vcd/resource_vcd_vapp_multi_vm_in_template_test.go

```diff
@@ -0,0 +1,157 @@
+// +build vapp vm ALL functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func TestAccVcdVAppMultiVmInTemplate(t *testing.T) {
+	var vapp govcd.VApp
+	var vm govcd.VM
+	vappName := t.Name()
+	vmName := t.Name() + "VM"
+	vmName2 := t.Name() + "VM2"
+	var params = StringMap{
+		"Org":                testConfig.VCD.Org,
+		"Vdc":                testConfig.VCD.Vdc,
+		"EdgeGateway":        testConfig.Networking.EdgeGateway,
+		"NetworkName":        "TestAccVcdVAppVmNet",
+		"Catalog":            testConfig.VCD.Catalog.Name,
+		"CatalogItemMultiVm": testConfig.VCD.Catalog.CatalogItemWithMultiVms,
+		"VmNameInTemplate":   testConfig.VCD.Catalog.VmName1,
+		"VmNameInTemplate2":  testConfig.VCD.Catalog.VmName2,
+		"VappName":           vappName,
+		"VmName":             vmName,
+		"VmName2":            vmName2,
+		"ComputerName":       vmName + "-unique",
+		"Tags":               "vapp vm",
+	}
+
+	configText := templateFill(testAccCheckVcdVAppVmMultiVmInTemplate, params)
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	if testConfig.VCD.Catalog.CatalogItemWithMultiVms == "" || testConfig.VCD.Catalog.VmName1 == "" || testConfig.VCD.Catalog.VmName2 == "" {
+		t.Skip("Variables catalogItemWithMultiVms, vmName1, vmName2  must be set to run multi VM in vApp template tests")
```

Comment:
This message doesn't match with what is said as comment in the sample configuration file.

Reply:
Is something I missing? ^^
![image](https://user-images.githubusercontent.com/41415459/81076278-2741a780-8ef4-11ea-9e5d-7f8f9c0741d1.png)

Reply:
I saw the message "# Name of VM in `catalogItemWithMultiVm` template or in `ovaMultiVmPath` if `catalogItemWithMultiVm` isn't provided. Default vmName `thirdVM` in default OVA."
It's because the sample config file for govcd is imported in vendor.

Reply:
Yes, this message from govcd.